src/lib/api/marketplace.ts

```diff
@@ -0,0 +1,42 @@
+import { gql } from 'apollo-boost'
+import { env } from 'decentraland-commons'
+import { createClient } from './graph'
+
+export const MARKETPLACE_URL = env.get('REACT_APP_MARKETPLACE_GRAPH_URL', '')
+const graphClient = createClient(MARKETPLACE_URL)
+
+const getSubdomainQuery = () => gql`
+  query getUserNames($owner: String) {
+    nfts(where: { owner: $owner, category: ens }) {
+      ens {
+        subdomain
+      }
+    }
+  }
+`
+
+type SubdomainTuple = {
+  ens: {
+    subdomain: string[]
+  }
+}
+
+type SubdomainQueryResult = {
+  nfts: SubdomainTuple[]
+}
```

Comment:
question for @cazala! for these `api` files we're keeping the types alongside the implementation right? there's no need to move this to it's own file